### ISSUE TITLE
New "How it works" pages

### ISSUE THREAD

**Issue by coreycaitlin** [18F/doi-extractives-data#2332]
After discussing where the new nonenergy, forestry, renewables, and other content belongs, we determined that we can break up the [production page](https://useiti.doi.gov/how-it-works/production/) into a new series of pages, which will be listed under a new header on the "How it works" page (which might be called "What natural resources are produced?" or similar). The new pages will include:

- [x] Fossil fuels (all content from https://useiti.doi.gov/how-it-works/production/#fossil-fuels)
- [x] Renewable energy (content from https://useiti.doi.gov/how-it-works/production/#renewable-energy + new addition) #2333 
- [x] Nonenergy minerals (content from https://useiti.doi.gov/how-it-works/production/#nonenergy-minerals + addition) #2314 
- [x] Forestry (all new content) #2313 

As we move, edit, and combine this content, we should keep an eye out for opportunities to reduce manual data updates and duplication.

**Comment by jamiealbrecht** [comment]
Hello! I've pulled in content from the nonenergy minerals addition on the how-it-works branch. Screenshots are attached to this comment. 

A little bit about what I did (in the order of the powerpoint documents):

**Overview:** The overview is great and I used most of it! I linked to the USGS page for each new mineral once hoping to not duplicate too much of their content. The USGS overview page lets people drill down if they want more information about a specific mineral. 

**Production:** We have production data for lead, zinc, gold, and silver in the explore data section. Do we want to include data for molybdenum there, too? The production stats, as custom content, will be hard to maintain over time and that information is also available via USGS. So, I didn't include them here, but we can talk about that!

**Industry overview** (Same as production) This information is all provided by the USGS. For simplicity, we should link to USGS and if people want more information, they can find it all there.

**Economic impact** A few questions: 
1. Are we going to start including content about imports and exports for each commodity?
Per our past user research, users are more interested in regional data, not by commodity, so I imagine no. However, I thought I should put the question here.
2. In the explore data section, we currently show a figure of jobs and wages for nonenergy minerals. Do we need to break that down further?
3. Are we going to start having a costs (environmental and the like) section for each commodity?
 
![screen shot 2017-09-18 at 9 19 39 am](https://user-images.githubusercontent.com/6954943/30552400-a3a2240e-9c52-11e7-9d62-736086555ffe.png)
![screen shot 2017-09-18 at 9 19 47 am](https://user-images.githubusercontent.com/6954943/30552404-a992d5de-9c52-11e7-9cc0-63befe29daa9.png)
